discrete_summary_checker/example/Bottle_Processing.ipynb

```diff
@@ -0,0 +1,1302 @@
+       "\n",
+       "  Oxygen Saturation, Garcia & Gordon [ml/l]  \\\n",
+       "0                                   7.53789   \n",
+       "1                                   7.72138   \n",
+       "2                                   7.72137   \n",
+       "3                                   7.72363   \n",
+       "4                                   7.72363   \n",
+       "\n",
+       "  Beam Attenuation, WET Labs C-Star [1/m]  \\\n",
+       "0                                  0.0459   \n",
+       "1                                  0.0702   \n",
+       "2                                  0.0696   \n",
+       "3                                  0.0594   \n",
+       "4                                  0.0593   \n",
+       "\n",
+       "  Beam Transmission, WET Labs C-Star [%]  \\\n",
+       "0                                98.8579   \n",
+       "1                                98.2607   \n",
+       "2                                98.2753   \n",
+       "3                                98.5278   \n",
+       "4                                98.5297   \n",
+       "\n",
+       "  Fluorescence, WET Labs ECO-AFL/FL [mg/m^3]  \n",
+       "0                               0.0273 (avg)  \n",
+       "1                               0.0097 (avg)  \n",
+       "2                              -0.0007 (avg)  \n",
+       "3                               0.0059 (avg)  \n",
+       "4                               0.0013 (avg)  \n",
+       "\n",
+       "[5 rows x 24 columns]"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Bottles.insert(0, \"Cruise\", \"SKQ202208S\")\n",
+    "Bottles.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c3ca249",
+   "metadata": {},
+   "source": [
+    "Save the results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "c7e3e851",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = \"Papa-09_SKQ202208S_CTD_Bottle_Data_2022-11-16_ACR.xlsx\"\n",
+    "filepath = \"/home/areed/Documents/OOI/reedan88/QAQC_Sandbox/Ship_data/data/Papa-09/\" + filename"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "50b0a602",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Bottles.to_excel(filepath, index=False)"
+   ]
+  }
+ ],
+ "metadata": {
+  "jupytext": {
+   "formats": "ipynb,auto:light"
+  },
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.12"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
OK (I used this notebook with Andrew to successfully compile Papa-09 .btl files.)